File: Feitos em C Sharp/1039.cs
using System;
namespace _1039
{
    class Program
    {
        static void Main(string[] args)
        {
            while (true)
            {
                string inp = Console.ReadLine();
                if (string.IsNullOrEmpty(inp))
                    break;
                string[] input = inp.Split();
                int raio1 = int.Parse(input[0]);
                int x1 = int.Parse(input[1]);
                int y1 = int.Parse(input[2]);
                int raio2 = int.Parse(input[3]);
                int x2 = int.Parse(input[4]);
                int y2 = int.Parse(input[5]);
                if (raio1 >= raio2 && Math.Pow((x1 - x2), 2) + Math.Pow((y1 - y2), 2) <= Math.Pow((raio1 - raio2), 2))
                    Console.WriteLine("RICO");
                else
                    Console.WriteLine("MORTO");
            }
        }
    }
}

File: Feitos em Python 3/1039.py
while True:
    try:
        ent = input().split()
        raio1 = int(ent[0]);
        x1 = int(ent[1]);
        y1 = int(ent[2]);
        raio2 = int(ent[3]);
        x2 = int(ent[4]);
        y2 = int(ent[5]);
        if (raio1 >= raio2 and ((x1 - x2) * (x1 - x2)) + ((y1 - y2) * (y1 - y2)) <= ((raio1 - raio2) * (raio1 - raio2))):
            print("RICO")
        else:
            print("MORTO");
    except EOFError:
        break